assets/js/blocks/active-filters/block.js

```diff
@@ -28,6 +32,10 @@ const ActiveFiltersBlock = ( {
 	attributes: blockAttributes,
 	isEditor = false,
 } ) => {
+	const filteringForPhpTemplate = getSetting(
```

Comment:
Shall we update this to use `getSettingWithCoercion` so its consistent with [other filters](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/blocks/attribute-filter/block.js#L66-L70)?